### PR TITLE
chore: release v0.1.364

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Comment by Lir10** [comment]
@tzvatot ptal

**Comment by machi1990** [comment]
Thanks @Lir10

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.364
+  - Add a new resource to OSL clusters/cluster_log
```
Comment (tzvatot) [inline CHANGES.md:7]: Usually we write down "Update model version v0.0.3xx" and then the details under. See v0.1.361 for example.

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (Lir10) [reply CHANGES.md:8]: done

**Review by tzvatot — APPROVED** [review]
LGTM

**Pull request merged**